app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,65 @@
+class RecipesController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_recipe, only: %i[destroy]
+
+  def index
+    @recipes = Recipe.where(user_id: current_user.id).includes(:recipe_foods)
+  end
+
+  def public
+    @recipes = Recipe.where(public: true).includes(:recipe_foods)
+  end
+
+  def new
+    recipe = Recipe.new
+    respond_to do |format|
+      format.html { render :new, locals: { recipe: } }
+    end
```

Comment:
- [ ] On line no. 16 and 51. I'm getting the error. Please fix that.

format.html { render :new, locals: { recipe: **recipe**} }


![Screenshot from 2023-02-10 21-37-28](https://user-images.githubusercontent.com/26767383/218143186-044156cf-2db2-47e6-a875-03a3133d3dec.png)